tools/cldr-apps/src/main/java/org/unicode/cldr/web/OutputFileManager.java

```diff
@@ -320,20 +323,26 @@ private boolean outputAllFiles(Writer out, File vetDataDir) {
         }
     }
 
+    public static Set<CLDRLocale> createVxmlLocaleSet() {
+        Set<CLDRLocale> set = new TreeSet<>(SurveyMain.getLocalesSet());
+        // skip "en" and "root", since they should never be changed by the Survey Tool
+        set.remove(CLDRLocale.getInstance("en"));
+        set.remove(CLDRLocale.getInstance(LocaleNames.ROOT));
+        // Remove "mul", "mul_ZZ", etc.
+        set.removeIf(loc -> loc.getBaseName().startsWith(LocaleNames.MUL));
```

Comment:
use SpecialLocales.java instead - in fact, if SpecialLocales.getType() returns non-null then skip. (sandbox (mul). algorithmic (sr_Latn) and other readonly.)

Reply:
I tried that and got different results. These locales get removed, so VXML isn't generated: ha_NE, sr_Latn, sr_Latn_BA, sr_Latn_ME, sr_Latn_XK, yo_BJ. They are all "algorithmic". Their files are present and not "empty" in baseline. If they don't get VXML generated, we'll get "Verification failure, file(s) present in baseline but not in vxml".

Would it be OK to merge this PR as-is, and then follow up with improvements?

Reply:
such files should be copied from baseline and not (re)generated.